packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -241,6 +244,81 @@ export const createRDSCluster = async (config: RDSConfig): Promise<ClusterInfo>
   }
 };
 
+/**
+ * Accesses the local RDS Aurora serverless V2 cluster with one DB instance using the given input configuration.
+ * @param config Configuration of the database cluster.
+ * @returns EndPoint address, port and database name of the accessed RDS cluster.
```

Comment:
nit: Update these doc comments to match updated method definition. You can just say returns the cluster configuration information.